c_emulator/riscv_sim.c

```diff
@@ -70,13 +76,17 @@ char *sig_file = NULL;
 uint64_t mem_sig_start = 0;
 uint64_t mem_sig_end = 0;
 int signature_granularity = 4;
+char *march_isa_string = NULL;
+int march_isa_int_ptr = 0;
 
 bool config_print_instr = true;
 bool config_print_reg = true;
 bool config_print_mem_access = true;
 bool config_print_platform = true;
 bool config_print_rvfi = false;
 
+int rv_cfg_c_ext_enable(char * isa_str, char * ext_pattern);
```

Comment:
If this is `static`, please make it `static`.

Reply:
Please provide documentation on the new function's contract.
It looks like this should return a `bool`?